### DISCUSSION TITLE
Add ability to group queries and reports into user defined groups

### DISCUSSION THREAD

**Discussion by tlammer** [specify/specify7#5528] [Feature Requests]
UMich requested being able to assign these by user group

**Comment by maxpatiiuk** [comment]
`...into user defined groups` and `...by user group` are different things.

Do they want an ability for a user to sort their own queries into some groups, or to share some queries to an entire user group?

Also, by user group, do you mean user type?

**Comment by maxpatiiuk** [comment]
From @grantfitzsimmons:

> The ability to group queries in folders would be nice.
> 
> **Requested By:** CSIRO

**Comment by danb213** [comment]
Hi, now that the statistics page has been added that runs off of queries having the ability to organise things in folders would be very helpful - especially for our admin account where we store the queries to pass out to the other users!
